tsconfig.base.json

```diff
@@ -35,12 +35,19 @@
         "libs/gi-dm-localization/src/index.ts"
       ],
       "@genshin-optimizer/gi-formula": ["libs/gi-formula/src/index.ts"],
+      "@genshin-optimizer/gi-frontend-gql": [
+        "libs/gi-frontend-gql/src/index.ts"
+      ],
       "@genshin-optimizer/gi-good": ["libs/gi-good/src/index.ts"],
       "@genshin-optimizer/gi-localization": [
         "libs/gi-localization/src/index.ts"
       ],
+      "@genshin-optimizer/gi-prisma-schema": [
```

Comment:
what is this and gi-frontend-gql? They don't seem to be in this PR

Reply:
oh weird, I was sure i removed these